DRAFT_Connectivity.Rmd

```diff
@@ -1,266 +1,280 @@
+
+
+## References
+Bakkestuen, V.; Venter, Z.; Ganerød, A.J.; Framstad, E. Delineation of Wetland Areas in South Norway from Sentinel-2 Imagery and LiDAR Using TensorFlow, U-Net, and Google Earth Engine. Remote Sens. 2023, 15, 1203. https://doi.org/10.3390/rs15051203
+
+Bakkestuen, V. et al. in prep. Comprehensive Mapping of Wetland Areas in Norway: An Integration of Sentinel-2 Imagery, LiDAR, and Deep Learning Using TensorFlow and Google Earth Engine
+
+Erikstad, L.; Simensen, T.; Bakkestuen, V.; Halvorsen, R. Index Measuring Land Use Intensity—A Gradient-Based Approach. Geomatics 2023, 3, 188-204. https://doi.org/10.3390/geomatics3010010
+
+Framstad, E., Eide, N.E., Eide, W., Klanderud, K., Kolstad, A., Töpper, J. & Vandvik, V. 2022. Vurdering av økologisk tilstand for fjell i Norge i 2021. NINA Rapport 2050. Norsk institutt for naturforskning.
+
+
+## Analyses {#analyses-con}
+### Data sets {#datasets-con}
+Two main data sets were included. 
+
+A. The infrastructure index (Erikstad et al. 2023) and; 
+
+B. the wetland model (Bakkestuen et al. 2023, Bakkestuen et al. _In prep._) 
+
+
+#### Data set A - Infrastructure index {#data-a-con}
+The infrastructure index utilized in this analysis was derived from data sources available through Google Earth Engine (GEE). The index quantifies the presence and distribution of key infrastructure elements such as roads, buildings, and related human-made structures across Norway. The data were available for the years 2003, 2013, and 2023, facilitating an assessment of how infrastructure has evolved in terms of both magnitude and distribution over time. It is important to note that this data forms a fundamental component in the calculation of the connectivity indicator.
+
+#### Data set B - Wetland model {#data-b-con}
+The wetland model used in this study was developed based on the integration of Sentinel-2 satellite imagery and LiDAR data, utilizing deep learning techniques. The model provides high-resolution information regarding the distribution and extent of wetland areas within southern Norway, specifically for the year 2020. By fusing spectral and elevation data, this model offers significant improvements over conventional, manually-digitized land cover maps. The wetland model is a critical component of the connectivity analysis, as it aids in the identification of ecologically relevant areas. The data set B for South-Norway is already accessible [here](https://kartkatalog.geonorge.no/metadata/aapen-vtmark-i-soer-norge-basert-p-natur-i-norge-nin-typologi/6b5a34f9-c23d-4cca-a51b-01251c217cb1).
+
+### Scaled indicator values {#scaled-values-con}
```

Comment:
There was some confusion about the terms thoughout. The reference values must be the connectivity without the infrastructure, and the scaling implies taking the quotient like I wrote here. But this scaling is not yet done #144